app/services/funding/update_trainee_summary_row_route_service.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Funding
+  class UpdateTraineeSummaryRowRouteService
+    include ServicePattern
+
+    ROUTE_TYPES = LeadPartnerTraineeSummariesImporter::ROUTE_TYPES.merge(
+      ProviderTraineeSummariesImporter::ROUTE_TYPES,
+    ).freeze
+
+    ACADEMIC_YEAR = "2024/25"
+
+    def initialize(academic_year = nil)
+      @academic_year = academic_year || ACADEMIC_YEAR
+    end
```

Comment:
Can we use the 

```
AcademicCycle.current.start_year
```

Reply:
It has to be in the format of `"2024/25"` from what I've seen locally with seed data.

Reply:
Could add a method to do the conversion